src/java/picard/analysis/CollectMultipleMetrics.java

```diff
@@ -197,6 +249,9 @@ public SinglePassSamProgram makeInstance(final String outbase, final File input,
     @Option(shortName = StandardOptionDefinitions.OUTPUT_SHORT_NAME, doc = "Base name of output files.")
     public String OUTPUT;
 
+    @Option(shortName="LEVEL", doc="The level(s) at which to accumulate metrics.")
+    public Set<MetricAccumulationLevel> METRIC_ACCUMULATION_LEVEL = new HashSet<MetricAccumulationLevel>();
```

Comment:
We may want different sub-programs to get called with different values of METRIC_ACCUMULATION_LEVEL, so it doesn't work to have it be a global command line argument. I'm not sure what the best solution to this is, though...